src/portal-components/DetailsPage.tsx

```diff
@@ -382,6 +384,10 @@ const SplitStringToComponent: React.FC<{
   const injectedProps = injectPropsIntoConfig(value, el, {
     ...deepCloneOfProps,
   })
+  if (overrideSqlSourceTable) {
```

Comment:
Note the new boolean parameter overrideSqlSourceTable in the RowToPropTransform.  (I made this sql spacing a little strange in case we need to track it down in the future!)